### ISSUE TITLE
terraform validate fails with providers requiring plugins

### ISSUE THREAD

**Issue by timw** [hashicorp/vscode-terraform#206]
When I execute the `Terraform: Validate` command, it executes `terraform validate` from outside the project directory (actually with `cwd='/')`.
When the Terraform project requires an external plugin (e.g. the `google` provider) then the validate fails. You can verify this in a terminal - it's not unique to this plugin and it appears that `terraform validate` has to be run from the project directory (where `terraform init` was run from) in order to succeed.

Sample log from my project:
```
2019-08-07 02:50:43:978 [INFO ] runner: Running terraform cwd='/' path='/usr/local/bin/terraform' (version: 0.12.6) args=[validate, -no-color, /Users/timw/projects/infra]
2019-08-07 02:50:44:023 [ERROR] runner: Running terraform failed: Error: Command failed: /usr/local/bin/terraform validate -no-color /Users/timw/projects/infra
2019-08-07 02:50:44:023 [ERROR] runner: 
2019-08-07 02:50:44:023 [ERROR] runner: Error: Could not satisfy plugin requirements
2019-08-07 02:50:44:023 [ERROR] runner: 
2019-08-07 02:50:44:023 [ERROR] runner: 
2019-08-07 02:50:44:023 [ERROR] runner: Plugin reinitialization required. Please run "terraform init".
```

**Comment by jpreese** [comment]
Same problem here. I was under the impression with 0.12, this step was no longer necessary

**Comment by RobM83** [comment]
Just here to confirm I got the same problem.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
